utils/cdi-provider/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-config-parent</artifactId>
+        <version>1.3.10-SNAPSHOT</version>
+        <relativePath>../../</relativePath>
+    </parent>
+    
+    <groupId>io.smallrye.config</groupId>
+    <artifactId>smallrye-config-cdi-provider</artifactId>
```

Comment:
Not sure this artifact name is descriptive enough, but I don't have a good alternative at the moment

Reply:
Not sure either. maybe just providers (so smallrye-config-providers) ?

Reply:
Done